skactiveml/pool/regression/_expected_model_change.py

```diff
@@ -0,0 +1,183 @@
+import numpy as np
+from sklearn import clone
+
+from skactiveml.base import (
+    SkactivemlRegressor,
+    SingleAnnotatorPoolQueryStrategy,
+)
+from skactiveml.pool.regression.utils._model_fitting import bootstrap_estimators
+from skactiveml.utils import (
+    check_type,
+    simple_batch,
+    check_scalar,
+    MISSING_LABEL,
+)
+from skactiveml.utils._validation import check_callable
+
+
+class ExpectedModelChange(SingleAnnotatorPoolQueryStrategy):
+    """Expected Model Change.
+
+    This class implements expected model change, an active learning
+    query strategy for linear regression.
+
+    Parameters
+    ----------
+    k_bootstraps: int, optional (default=3)
+        The number of bootstraps used to estimate the true model.
+    n_train: int or float, optional (default=0.5)
+        The size of a bootstrap compared to the training data.
+    ord: int or string (default=2)
+        The Norm to measure the gradient. Argument will be passed to
+        `np.linalg.norm`.
+    feature_map: callable
+        The feature map of the linear regressor. Takes in the feature data. Must
+        output a np.array of dimension 2.
+    missing_label : scalar or string or np.nan or None, default=np.nan
```

Comment:
if you change the default value for missing label in `skactiveml.utils` then you will have to change the documentation all over the place, as it won't be np.nan. 

I would suggest something like:

```suggestion
    missing_label : scalar or string or np.nan or None, default=skactiveml.utils.MISSING_LABEL
```